packages/@aws-cdk/aws-kinesis/lib/stream.ts

```diff
@@ -132,7 +139,11 @@ abstract class StreamBase extends Resource implements IStream {
    * contents of the stream will also be granted.
    */
   public grantWrite(grantee: iam.IGrantable) {
-    const ret = this.grant(grantee, 'kinesis:DescribeStream', 'kinesis:PutRecord', 'kinesis:PutRecords');
+    const ret = this.grant(
+      grantee,
+      'kinesis:ListShards',
```

Comment:
Is `ListShards` needed for write? I know `DescribeStream` was here but do we know why?
Also - are we sure getting rid of it doesn't break anything? doesn't that mean that users who previously could describe a stream with just the write permissions won't be able to now?

Im ok with that, we just need to publish this as a breaking change in that case.

Reply:
yes it is, I was advised to add it by @yasemin-amzn to align with the Kinesis Producer Library.

I included the rationale already for breaking change in the commit message... is there anything to do here?

Reply:
Whoops - didn't notice the breaking changes notice. All good 👍